src/openrct2/park/ParkFile.cpp

```diff
@@ -941,7 +941,7 @@ namespace OpenRCT2
 
                     if (cs.GetMode() == OrcaStream::Mode::READING)
                     {
-                        OpenRCT2::GetContext()->GetGameState()->InitAll(gMapSize);
+                        OpenRCT2::GetContext()->GetGameState()->InitAll(gMapSize, true);
```

Comment:
Why is this necessary? `ReadWriteClimateChunk` is done after this and that will overwrite the climate anyway.

Reply:
I wasn't aware of that. So then, the only thing that needs to change is the 2 removed calls of the reset function?

Reply:
Yes